### PR TITLE
Fixes #4811 - Added instructions to install PowerShellGet without existing version installed

### LINKED ISSUE #4811: PackageManagement MSI No Longer Supported
Please update this page to reflect new instructions for how to install for PowerShell 3.0 and 4.0 since the PackageManagement MSI is no longer supported.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c6bc5a9a-76fe-42f6-f11f-17e59af93f4f
* Version Independent ID: 8d0ef725-578d-bd63-8b4a-5a14a54f7dd4
* Content: [Installing PowerShellGet](https://docs.microsoft.com/en-us/powershell/gallery/installing-psget#feedback)
* Content Source: [gallery/installing-psget.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/gallery/installing-psget.md)
* Product: **powershell**
* Technology: **powershell-gallery**
* GitHub Login: @SydneyhSmith
* Microsoft Alias: **sysmith**

### PR DESCRIPTION
**Pull request by davidsmatlak**
- Fixes [AB#1601532](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1601532)
- Fixes #4811

### CODE REVIEW

File: gallery/installing-psget.md [hunk 1]
```diff
@@ -60,3 +64,23 @@ Exit
    Copy-Item "C:\LocalFolder\PowerShellGet\*" "$env:ProgramFiles\WindowsPowerShell\Modules\PowerShellGet\" -Recurse -Force
    Copy-Item "C:\LocalFolder\PackageManagement\*" "$env:ProgramFiles\WindowsPowerShell\Modules\PackageManagement\" -Recurse -Force
    ```
+
+#### Computers without PowerShellGet
+
+For a computer that doesn't have any version of **PowerShellGet** installed, you can download it from the [PowerShell Gallery](https://www.powershellgallery.com/).
```
Comment (DCtheGeek) [inline gallery/installing-psget.md]: Let's rephrase this a bit.  That first sentence by itself makes it sound like a PS 3/4 computer without PowerShellGet can just get it from the Gallery.

File: gallery/installing-psget.md [hunk 2]
```diff
@@ -60,3 +64,23 @@ Exit
...
+You'll need a computer that has **PowerShellGet** installed.
+
+1. From a computer that has **PowerShellGet** installed, download the current version of
+   **PowerShellGet**. Two folders are downloaded: **PowerShellGet** and **PackageManagement**. Each
+   folder contains a subfolder with a version number.
+
+   For more information, see [Save-Module](/powershell/module/PowershellGet/Save-Module).
```
Comment (DCtheGeek) [inline gallery/installing-psget.md]: Move this line to under the code snippet and add a little context as to what `Save-Module` does and why a user might want additional information about it.

File: gallery/installing-psget.md (hunk 1 above)
Comment (davidsmatlak) [reply gallery/installing-psget.md]: Thanks for the feedback. Fixed.

File: gallery/installing-psget.md (hunk 2 above)
Comment (davidsmatlak) [reply gallery/installing-psget.md]: Moved this to the H3 section because both sets of instructions use `Save-Module`.

**Pull request merged**